packages/react-core/src/components/Alert/Alert.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+
+import AlertIcon from './AlertIcon';
+import AlertBody from './AlertBody';
+import AlertAction from './AlertAction';
+import Button from '../Button/Button';
+
+export const alertType = ['success', 'danger', 'warning', 'info'];
+
+const propTypes = {
+  type: PropTypes.oneOf(alertType).isRequired,
+  action: PropTypes.shape(Button.propTypes),
+  ariaLabel: PropTypes.string,
+  isAriaHidden: PropTypes.bool,
+  title: PropTypes.string,
+  children: PropTypes.node,
+  className: PropTypes.string
+};
+
+const defaultProps = {
+  ariaLabel: '',
+  isAriaHidden: true,
+  action: null,
+  title: '',
+  children: '',
+  className: ''
+};
+
+const Alert = ({
+  type,
+  ariaLabel,
+  action,
+  title,
+  isAriaHidden,
+  children,
+  className,
+  ...props
+}) => (
+  <div
+    {...props}
+    className={css(
+      styles.alert,
+      getModifier(styles, type, styles.modifiers.info),
+      className
+    )}
+    aria-label={ariaLabel}
+  >
+    <AlertIcon type={type} isAriaHidden={isAriaHidden} />
```

Comment:
Fine to leave, but I am not really sure we need these as separate components.  We are not exporting them, and they are single HTML elements.  Icon I can see adding some logic through with type so it might make sense for it to have it's own, but that should still only be a simiple lookup map. Keeping them in this file will simplify reading and testing this component.

Reply:
I think bigger granularity for these components is a plus, because when there is some change you can focus to just that part and not to focus on not breaking oter parts of alert. On other side I may have overcomplicated it when I broke body into title and rest. So I guess 3 components - `Icon`, `Body`, `Action` is better.